docs/keyword-replacement.md

```diff
@@ -26,14 +26,23 @@ To use the keyword replacement, the `AUTH0_KEYWORD_REPLACEMENT_MAPPINGS` configu
 
 ```yaml
 tenant:
-  friendly_name: My ##ENVIRONMENT## tenant
-  allowed_logout_urls: @@ALLOWED_LOGOUTS@@
+  friendly_name: "##ENVIRONMENT## tenant"
```

Comment:
Adding an example with wrapped quotes shows how to implement string replace keyword when there is no preceding string to the keyword.